website/docs/index.html.markdown

```diff
@@ -8,15 +8,52 @@ description: |-
 
 # Lynx Provider
 
-The Lynx Provider can be used to configure infrastructure in [Lynx Terraform backend](https://github.com/Clivern/Lynx/blob/main/README.md) using the Lynx API's. Documentation regarding the [Data Sources](/docs/configuration/data-sources.html) and [Resources](/docs/configuration/resources.html) supported by the Lynx Provider can be found in the navigation to the left.
+The Lynx Provider can be used to configure infrastructure in [Lynx Terraform backend](https://github.com/Clivern/Lynx/blob/main/README.md) using the Lynx API's. Documentation regarding the Data Sources and Resources supported by the Lynx Provider can be found in the navigation to the left.
 
-To learn the basics of Terraform using this provider, follow the
-hands-on [get started tutorials](https://github.com/Clivern/Lynx/blob/main/README.md).
+To learn the basics of Terraform using this provider, follow the hands-on [Get started tutorial](https://github.com/Clivern/Lynx/blob/main/README.md).
 
-## Authenticating to Azure
+## Authenticating to Lynx
 
-## Example Usage
+### Example Usage
 
+```terraform
+# Configure the Lynx Provider 
+provider "lynx" {
+  api_url = "http://localhost:4000/api/v1"
+  api_key = "~api key here~"
+}
+# Create the resources
+```
+
+~> **IMPORTANT** Hard-coding yourLynx programmatic API key pair into a Terraform configuration is not recommended.
+Consider the risks, especially the inadvertent submission of a configuration file containing secrets to a public repository.
+
+### Environment Variables
+
+You can also provide your credentials via the environment variables, 
+Lynx programmatic API key pair respectively:
+
+Usage (prefix the export commands with a space to avoid the keys being recorded in OS history):
+
+```shell
+$ export LYNX_API_URL="xxxx"
```

Comment:
@Clivern  Coulde you please double-check this Environments variables?

Reply:
Yes that should work 👍 , https://github.com/Clivern/terraform-provider-lynx/blob/main/internal/provider/provider.go#L92-L93